test/check-machines-storage-pools

```diff
@@ -748,6 +748,239 @@ class TestMachinesStoragePools(VirtualMachinesCase):
             remove=True,
         ).execute()
 
+    def testStoragePoolsDeletion(self):
+        b = self.browser
+        m = self.machine
+
+        # Prepare some storage pools dependencies
+        # NFS
+        m.execute("if selinuxenabled 2>/dev/null; then setsebool -P virt_use_nfs 1; fi")
+        self.restore_file("/etc/exports")
+        nfs_pool = os.path.join(self.vm_tmpdir, "nfs_pool")
+        mnt_exports = os.path.join(self.vm_tmpdir, "mnt_exports")
+        m.execute(f"mkdir {nfs_pool} {mnt_exports} && echo '{mnt_exports} 127.0.0.1/24(rw,sync,no_root_squash,no_subtree_check,fsid=0)' > /etc/exports")
```

Comment:
The restore_file and that echo can be done more elegantly (and robustly) with

    self.write_file("/etc/exports", "...content...", append=True, post_restore_action="systemctl restart nfs-server")

This will also restart nfs-server after restoring.

Reply:
OK, re-push